### ISSUE TITLE
Deleted Files

### ISSUE THREAD

**Issue by jrdevlacco** [Len-Stevens/Python-Antivirus#15]
Where does the deleted file goes? I choose delete the file option and then i couldn't find the file anywhere.

Also, after clicking delete the file button, there are 2 messages showing up. First, succesfully deleted, and then file is not deleted, and succesfully deleted message again.

Thanks for your help!

**Comment by raphael069** [comment]
Yes so the first issue is already fixed in a version i did´nt release yet and when you click the delete file  button It will completly delete the file I think.

**Comment by nagpalansh27** [comment]
could you please share the version with me where the file deletion works properly?

**Comment by raphael069** [comment]
Its on my profile if you need it 
i think i uploaded it already
